cmd/bblfsh-sdk/cmd/fixtures.go

```diff
@@ -26,7 +26,7 @@ type FixturesCommand struct {
 	Endpoint  string `long:"endpoint" short:"e" default:"localhost:9432" description:"Endpoint of the gRPC server to use"`
 	ExtNative string `long:"extnative" short:"n" default:"native" description:"File extension for native files"`
```

Comment:
```suggestion
* All the integer literals in Python: `//*[@NumType='bool']`
```

this is just a test for new GH feature of "suggestions". Please discard